README.rst

```diff
@@ -26,10 +26,6 @@
     :target: https://codeclimate.com/github/melexis/sphinx-coverity-extension
     :alt: Issue Count
 
-.. image:: https://requires.io/github/melexis/sphinx-coverity-extension/requirements.svg?branch=master
```

Comment:
why are we removing this badge? Also commit message doesn't say...

Reply:
The badge no longer works. Requires.io shut down on September 30th, 2022.